source/integrate-with-integration-environment/authenticate-your-user.html.md.erb

```diff
@@ -211,16 +211,16 @@ Location: https://YOUR_REDIRECT_URI?error=invalid_request
+|-------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `unauthorized_client`                     | In exceptional circumstances, such as a security incident, One Login may prevent users from logging in to your service. In this event the error code `unauthorized_client` will be returned with the error description `client deactivated`. When your service receives this error you should show the user a custom error page to explain that they cannot use your service at the moment and should try again later.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| `unauthorized_client`                     | In rare circumstances, such as a security incident, One Login may prevent users from logging in to your service. If this happens, the error code `unauthorized_client` will be returned with the error description `client deactivated`. When your service receives this error, you must show the user a custom error page to explain that they cannot use your service at the moment and should try again later.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Some small suggestions for clarity. I'm not sure on 'must' vs. 'should' - 'should' implies that they don't have to show a custom error page, but then what happens if they don't?

Reply:
I've committed this change. If they don't it would likely be caught up in their generic error handling, which would probably have similar copy. No harm in a stronger requirement, however. I'm sure RPs will ignore it if they don't think it's necessary...